bundles/net.tourbook/src/net/tourbook/messages.properties

```diff
@@ -2788,84 +2786,87 @@ TourGeoFilter_Loader_LoadingError = Loading error:
-Tour_Action_auto_move_sliders_when_zoomed      = Auto-&Move vertical Sliders
-Tour_Action_auto_zoom_to_slider_position       = Auto-&Zoom to Slider Position
-Tour_Action_graph_altimeter_tooltip            = Show/hide altimeter
-Tour_Action_graph_altitude_tooltip             = Show/hide elevation
-Tour_Action_graph_analyzer_tooltip             = Graph Analyzer
-Tour_Action_graph_cadence_tooltip              = Show/hide cadence
-Tour_Action_graph_gradient_tooltip             = Show/hide gradient
-Tour_Action_graph_heartbeat_tooltip            = Show/hide heartbeat
-Tour_Action_graph_pace_tooltip                 = Show/hide pace
-Tour_Action_graph_power_tooltip                = Show/hide power
-Tour_Action_graph_speed_tooltip                = Show/hide speed
-Tour_Action_graph_temperature_tooltip          = Show/hide temperature
-Tour_Action_graph_tour_compare_tooltip         = Tour Compare Result
-Tour_Action_scroll_zoomed_chart                = Zoomed Chart can be scrolled
-Tour_Action_show_distance_on_x_axis            = X-Axis Distance
-Tour_Action_show_distance_on_x_axis_tooltip    = Show distance on the x-axis
-Tour_Action_show_start_time_on_x_axis          = Show Start Time on X-Axis
-Tour_Action_show_time_on_x_axis                = X-Axis Time
-Tour_Action_show_time_on_x_axis_tooltip        = Show time on the x-axis
+Tour_Action_Marker_SetVisible                   = Set Marker &Visible 
+Tour_Action_MultiplyCaloriesBy1000              = &Multiply calories value by 1000...
+Tour_Action_MultiplyCaloriesBy1000_Apply        = &Multiply Calories Value
+Tour_Action_MultiplyCaloriesBy1000_Message      = The calories value will be multiplied by 1000 for all ({0}) selected tours\n\
+                                                  \n\
+                                                  \n\
+                                                  THIS ACTION CANNOT BE REVERTED\n\
+                                                  \n\
+                                                  \u2192 Create a backup of the tour data before doing this action !
+Tour_Action_MultiplyCaloriesBy1000_Title        = Multiply Calories Value
+Tour_Action_RetrieveWeatherData                 = Retrieve &Weather Data...
+Tour_Action_RunDyn_StanceTimeBalance_Tooltip    = Show/hide stance time balance
+Tour_Action_RunDyn_StanceTime_Tooltip           = Show/hide stance time 
+Tour_Action_RunDyn_StepLength_Tooltip           = Show/hide step length
+Tour_Action_RunDyn_VerticalOscillation_Tooltip  = Show/hide vertical oscillation
+Tour_Action_RunDyn_VerticalRatio_Tooltip        = Show/hide vertical ratio
+Tour_Action_Select_Inbetween_Timeslices         = Select in-between time slices
+Tour_Action_Select_Inbetween_Timeslices_Tooltip = If checked, selects all the time slices in the Tour Editor View between the left and right sliders. Otherwise, selects only the current slider's \
+                                                  time slice.
```

Comment:
"If checked, selects all the time slices in the Tour Editor View between the left and right sliders. Otherwise, selects only the current slider's time slice."

"If checked, selects all the time slices in the Tour Editor View between the left and right sliders. Otherwise **it** selects only the current slider's time slice."

Wording? You are the english/french man :-)

Reply:
Will fix!